### DISCUSSION TITLE
allow the user to choose the branch/release of the plugin repository

### DISCUSSION THREAD

**Discussion by tareknaser** [coffee-tools/coffee#130] [Ideas]
Based on our earlier discussion, it was suggested to incorporate a new feature into upgrade command #43. 

Specifically, the proposed idea involves the addition of an option to select a specific branch or release of a remote repository, which can be specified by passing a flag. An example of the syntax for using this feature would be
`coffee upgrade folgore --release 2.1`

*It is important to note that this proposal assumes the presence of only one plugin in each remote repository.*

I think such feature would improve the overall user experience.

**Comment by vincenzopalazzo** [comment]
I agree, we should start experimenting with it, I will move this as an issue :)
